src/journal/journal-send.c

```diff
@@ -158,6 +165,8 @@ _printf_(1, 0) static int fill_iovec_sprintf(const char *format, va_list ap, int
 
                 VA_FORMAT_ADVANCE(format, ap);
 
+                (void) strstrip(buffer); /* strip trailing whitespace, keep prefixing whitespace */
```

Comment:
I think we shouldn't change the behaviour of the `sd_journal_send`
I.e. `sd_journal_send("MESSAGE=BOO", "VALUE=%s", " ", NULL)` should produce:
```
        "MESSAGE" : "BOO",
        "VALUE" : " ",
```
instead of
```
        "MESSAGE" : "BOO",
        "VALUE" : "",
```